src/index.tsx

```diff
@@ -270,7 +270,9 @@ const OtpInputs = forwardRef<OtpInputsRef, Props>(
             ref={inputs.current[inputIndex]}
             secureTextEntry={secureTextEntry}
             selectTextOnFocus={selectTextOnFocus}
+            accessible={true}
```

Comment:
Just `accessible` is enough I think :)
```suggestion
            accessible
```

Reply:
Tried, unfortunately - no, current React Native Android implementation does not correctly utilises resource-id generation for TextInput, and only addition of `accessibilityLabel` will create `content-desc` attribute in native elements which can be found using `$('~testID')` locator strategy via Appium

Reply:
Used to strict typescript and eslint checks, this should also work